src/Dropdown/Dropdown.js

```diff
@@ -285,8 +284,18 @@ Dropdown.Header = ({ title, children }) => (
 Dropdown.Body = createComponent({
   name: 'DropdownBody',
   as: Box,
-  style: css`
-    padding: 1rem;
+});
+
+Dropdown.Section = createComponent({
+  name: 'DropdownSection',
+  as: Flex,
+  style: ({ theme }) => css`
+    padding: 16px 8px 8px 8px;
+    flex-direction: column;
+
+    &:not(:last-of-type) {
+      border-bottom: 1px solid ${theme.colors.greyLight};
+    }
```

Comment:
@mathewmorris What necessitated the addition of this `Section` component? It seems to me like `Dropdown.Body` had the necessary padding already, which was removed. Can we stick as close the original implementation as possible and only add/remove things surgically to match the new Figma requirements? That would be greatly appreciated! 😄 💪 👍

Reply:
What do you recommend we do with the separators? Right now they are spanning across 100% of the dropdown using the `Dropdown.Section`. I'm not sure how keeping the padding in the body would allow that to happen.
That was the main reason I added `Dropdown.Section`.

Reply:
@mathewmorris I don't see a separator in the molekule preview for dropdown sections... 🤔

Reply:
@mathewmorris @erikshestopal I'm also not seeing the icon usage in the deploy preview, but I see it in the github diff...

Reply:
Yeah, I'm not sure what's going on with the preview. I'm also seeing everything locally.

Reply:
I'm seeing the changes updated now on the preview